### PR TITLE
[RFR] Adding pytest-repeat to requirements

### PR DESCRIPTION
**Pull request by jan-zmeskal**
__Extending__ requirements of the FW. Here's a rationale why I think we could benefit from including this package:

__Use case 1__: You want to really verify your new test case and make sure it is solid and does not get broken by some random occurrences. You can run it 10 times with:
`pytest test_stuff.py --count 10`

__Use case 2__: You know that some test case fails in very specific condition, but you don't know what precisely is that and want to reproduce the failure. With this package you can do:
`pytest --count=1000 -x test_stuff.py`

I know I can just install this locally and be happy with it. However including this in virtualenv would make it easy to do the same thing on Jenkins. Also, the package has like 4KB, so it's not like we're slowing down the installation.

pytest-repeat link: https://pypi.python.org/pypi/pytest-repeat#downloads

Addendum: 
The repeating is achieved by parametrization. Example here:
`miq-runtest cfme/tests/test_login.py::test_login --count=5`

> cfme/tests/test_login.py::test_login[1/5-ViaUI-click_on_login] PASSED
> cfme/tests/test_login.py::test_login[1/5-ViaUI-press_enter_after_password] PASSED
> cfme/tests/test_login.py::test_login[1/5-ViaUI-_js_auth_fn] PASSED
> cfme/tests/test_login.py::test_login[1/5-ViaSSUI-click_on_login] PASSED
> cfme/tests/test_login.py::test_login[1/5-ViaSSUI-press_enter_after_password] PASSED
> cfme/tests/test_login.py::test_login[2/5-ViaUI-click_on_login] PASSED
> cfme/tests/test_login.py::test_login[2/5-ViaUI-press_enter_after_password] PASSED
> cfme/tests/test_login.py::test_login[2/5-ViaUI-_js_auth_fn] PASSED
> cfme/tests/test_login.py::test_login[2/5-ViaSSUI-click_on_login] PASSED
> cfme/tests/test_login.py::test_login[2/5-ViaSSUI-press_enter_after_password] PASSED
> cfme/tests/test_login.py::test_login[3/5-ViaUI-click_on_login] PASSED
> cfme/tests/test_login.py::test_login[3/5-ViaUI-press_enter_after_password] PASSED
> cfme/tests/test_login.py::test_login[3/5-ViaUI-_js_auth_fn] PASSED
> cfme/tests/test_login.py::test_login[3/5-ViaSSUI-click_on_login] PASSED
> cfme/tests/test_login.py::test_login[3/5-ViaSSUI-press_enter_after_password] PASSED
> cfme/tests/test_login.py::test_login[4/5-ViaUI-click_on_login] PASSED
> cfme/tests/test_login.py::test_login[4/5-ViaUI-press_enter_after_password] PASSED
> cfme/tests/test_login.py::test_login[4/5-ViaUI-_js_auth_fn] PASSED
> cfme/tests/test_login.py::test_login[4/5-ViaSSUI-click_on_login] PASSED
> cfme/tests/test_login.py::test_login[4/5-ViaSSUI-press_enter_after_password] PASSED
> cfme/tests/test_login.py::test_login[5/5-ViaUI-click_on_login] PASSED
> cfme/tests/test_login.py::test_login[5/5-ViaUI-press_enter_after_password] PASSED
> cfme/tests/test_login.py::test_login[5/5-ViaUI-_js_auth_fn] PASSED
> cfme/tests/test_login.py::test_login[5/5-ViaSSUI-click_on_login] PASSED
> cfme/tests/test_login.py::test_login[5/5-ViaSSUI-press_enter_after_password] PASSED

PRT fails: provider_crud failed for some providers in PRT, which seems pretty random and unrealted.

### CODE REVIEW

File: requirements/frozen.txt [hunk 1]
```diff
@@ -132,6 +132,7 @@ Pygments==2.2.0
 PyJWT==1.5.0
 pyparsing==2.2.0
 pytest==3.2.0
+pytest-repeat==0.4.1
```
Comment (RonnyPfannschmidt) [inline requirements/frozen.txt]: **required** a unpinned dependency in the requirement template

Comment (jan-zmeskal) [reply requirements/frozen.txt]: @RonnyPfannschmidt could you pls elaborate?

Comment (RonnyPfannschmidt) [reply requirements/frozen.txt]: the frozen files are regenerated from the requirement templates on occasion freezing new versions of everything, leaving new packages out of that means we open up for a regression

Comment (jan-zmeskal) [reply requirements/frozen.txt]: I see! Thank you for an explanation, I will edit the accordingly

**Comment by quarckster** [comment]
I like the idea, but it's not mandatory for the framework execution. I'm not sure if it should be included in that case.

**Comment by jan-zmeskal** [comment]
@quarckster It is true that this is definitely not required by the FW, I just thought other people could benefit from this functionality as well. But if we try to keep the number of installed packages to minimum, that's okay. I can always deploy it only in our env.

**Pull request closed without merging**